x-pack/plugins/observability/public/pages/slos/helpers/sort_slos.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { SLO } from '../../../typings';
+import { SortType } from '../components/slo_list_search_filter_sort_bar';
+import { getSloDifference } from './get_slo_difference';
+
+export function sortSlos(sort: SortType | undefined) {
```

Comment:
I guess the sort always uses the Asc direction at the moment. I guess we can keep it like this until we have the API with support for sorting/filtering